src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -58,6 +58,30 @@ public static void setField(
         }
   }
 
+  /**
+   * Changes a given static field of a class via reflection, bypassing the
```

Comment:
:pick: "Set" instead of "Changes"

Reply:
@svorenova if you follow Romain's suggestion, maybe also adapt the documentation of the other methods.